elements/rh-audio-player/rh-audio-player-about.css

```diff
@@ -0,0 +1,39 @@
+.media-info {
+  display: var(--_about-mediatitle-display, none);
```

Comment:
Nope. The design specifies that if the player is not in full mode, that instead of the about panel heading, the title of the media will display, so I needed a away to make that happen via CSS.